### DISCUSSION TITLE
24FC9F77-7DD1-43CB-995A-C3673A51AFFC

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#184] [General]
# Red Giant Sun

A depiction of the Solar System 7.9 billion years from now, when the Sun has reached its maximum size as a red giant. Taken from Celestia Origin, with a red giant model & texture by nussun added.

https://celestia.mobi/addon?item=24FC9F77-7DD1-43CB-995A-C3673A51AFFC&lang=es

**Comment by SiriusBislifeless** [comment]
This isn't working on Celestia of Microsoft Store

**Reply by DaveBowman2001** [reply]
Hello @SiriusBislifeless, I've asked for help regarding this matter on Discord, and here are their responses:
![IMG_20240705_225333](https://github.com/celestiamobile/comments/assets/95486841/6902f4af-3d07-44df-8f49-f37e8da4c720)
![IMG_20240705_225534](https://github.com/celestiamobile/comments/assets/95486841/19cca7e5-78a3-4c58-826e-d0c12c2c5b36)
You can try updating your MS-Celestia to the latest version to fix the issue about downloading addons

**Reply by SiriusBislifeless** [reply]
Yeah, it seems updating was the solution, thx man

**Comment by Mrcheemsgorebox** [comment]
Im actually  kinda pissed it doesn't work on mobile or isn't showing but why?

**Reply by zjx29rjhdh-pixel** [reply]
Your device’s corrupted

**Comment by SevenSpheres** [comment]
To the people who say this isn't working - are you expecting the default Solar System to be modified? The Solar System with a red giant Sun is a separate system named "Red Giant Sun". On mobile "Open in Celestia" should take you there I think?

**Reply by SiriusBislifeless** [reply]
My bad, i was expecting the default Solar System to be modified, thankz

**Reply by zjx29rjhdh-pixel** [reply]
Thanks , I was expecting the default Solar System to be modified.

**Comment by vinotintocorp-create** [comment]
We Need to Fix it to have a year when sun grows up animated, (1 BY - 7.8 BY) in the future.
Sun Solar Flare fire Animated 3D Blending harder $1,000,000 Budget.
